pkg/builder/controller.go

```diff
@@ -319,7 +318,6 @@ func (blder *Builder) doController(r reconcile.Reconciler) error {
 			log := log
 			if req != nil {
 				log = log.WithValues(
-					gvk.Kind, klog.KRef(req.Namespace, req.Name),
 					"namespace", req.Namespace, "name", req.Name,
```

Comment:
```suggestion
					"namespace", req.Namespace, "name", req.Name,
```

We should probably remove this line instead?

Reply:
@vincepri thanks for your time to review this.

If I understand correctly, your proposition would result in something like this:

```
level=INFO msg="creating certificate request" controller=certificate controllerGroup=example.com controllerKind=Certificate Certificate="{certificate-sample appl-k8s-e2etests1-e1}" reconcileID=8e0379c3-ec61-42a7-9bb1-09b19f57bfb6 rev=1
```

We would also have to remove this [line](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/builder/controller.go#L315). Otherwise we would still have redundant information in the log line. This would produce:


```
level=INFO msg="creating certificate request" controller=certificate controllerGroup=example.com Certificate="{certificate-sample appl-k8s-e2etests1-e1}" reconcileID=8e0379c3-ec61-42a7-9bb1-09b19f57bfb6 rev=1
```

In comparison my change:

```
level=INFO msg="creating certificate request" controller=certificate controllerGroup=example.com controllerKind=Certificate namespace=appl-k8s-e2etests1-e1 name=certificate-sample reconcileID=8e0379c3-ec61-42a7-9bb1-09b19f57bfb6 rev=1
```

If you are using indexing software like Splunk or Elastic Search, it would be harder to write queries for your proposition, since the Kind (`Certificate` in our the case above) is now a key in the structured log. If you are reconciling for different kinds, we would have a key, that can change for different log entries.